Linux/Car_Client/chronoscomm.cpp

```diff
@@ -46,6 +47,8 @@ ChronosComm::ChronosComm(QObject *parent) : QObject(parent)
             this, SLOT(tcpInputDisconnected()));
     connect(mTcpSocket, SIGNAL(error(QAbstractSocket::SocketError)),
             this, SLOT(tcpInputError(QAbstractSocket::SocketError)));
+
+    mGpioControl->setGPIO_Out(4);
```

Comment:
A check should be made when receiving ACCM that this pin is configured as an output (otherwise configure it)

Reply:
Also, the pin number should be replaced with a suitable #define

Reply:
Changed so pin is unset and then set when receiving ACCM and added #defines